rts/lra/lra-test/src/main/java/io/narayana/lra/participant/api/StandardController.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package io.narayana.lra.participant.api;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import java.util.logging.Logger;
+
+import static io.narayana.lra.participant.api.ActivityController.ACTIVITIES_PATH;
+import static io.narayana.lra.participant.api.StandardController.ACTIVITIES_PATH3;
```

Comment:
Is this a cyclic import?

Reply:
No. I could do

> @Path(StandardController.ACTIVITIES_PATH2)

instead but I think the static import is neater

Reply:
OK, I disagree (slightly) but not holding the PR up for this one.

Reply:
Okay but since I am making a number of other changes I will go with StandardController.ACTIVITIES_PATH2 (since the recommendation is not overuse static imports)